redbot/core/bank.py

```diff
@@ -1085,3 +1120,41 @@ async def wrapped(*args, **kwargs):
             return coro_or_command
 
     return deco
+
+
+class BankTransferInformation:
+    def __init__(
+        self,
+        transfer_amount: int,
+        sender_id: int,
+        recipient_id: int,
+        guild_id: int,
+        sender_new_balance: int,
+        recipient_new_balance: int,
+    ):
+        self.transfer_amount = transfer_amount
+        self.sender_id = sender_id
+        self.recipient_id = recipient_id
+        self.guild_id = guild_id
+        self.sender_new_balance = sender_new_balance
+        self.recipient_new_balance = recipient_new_balance
+
+
+class BankSetBalanceInformation:
+    def __init__(
+        self,
+        recipient_id: int,
+        guild_id: int,
+        recipient_old_balance: int,
+        recipient_new_balance: int,
+    ):
+        self.recipient_id = recipient_id
+        self.guild_id = guild_id
+        self.recipient_old_balance = recipient_old_balance
+        self.recipient_new_balance = recipient_new_balance
+
+
+class BankPruneInformation:
```

Comment:
The type annotation for `pruned_users` should be `Union[List[int], Dict[str, List[int]]]` instead of `Union(List[int], Dict[str, List[int]])`